lib/signs_ui/application.ex

```diff
@@ -4,24 +4,30 @@ defmodule SignsUi.Application do
   alias SignsUi.Utilities.Config
 
   def start(_type, _args) do
-    import Supervisor.Spec
-
     set_runtime_config()
 
-    # Define workers and child supervisors to be supervised
     children = [
-      # Start the endpoint when the application starts
-      supervisor(SignsUiWeb.Endpoint, []),
-      worker(SignsUI.Signs.State, [[]]),
-      worker(SignsUi.Signs.Messages, [[name: SignsUi.Signs.Messages]])
-      # Start your own worker by calling: SignsUi.Worker.start_link(arg1, arg2, arg3)
-      # worker(SignsUi.Worker, [arg1, arg2, arg3]),
+      %{
```

Comment:
You can use the simple shorthand for children:

```ex
children = [
  SignsUiWeb.Endpoint,
  SignsUI.Signs.State,
  {SignsUi.Signs.Messages, [[name: SignsUi.Signs.Messages]]},
  SignsUI.Signs.Expiration
]
```

You'll need to add `use GenServer` to your module, which it definitely should have, since it's a GenServer.

Reply:
Done